src/Calculations/Addition.cs

```diff
@@ -12,16 +11,21 @@ public Addition(IUserConsoleInputValidation validateInput)
     public double FirstNumber { get; private set; }
     public double SecondNumber { get; private set; }
 
-    public void AddWorkFlow()
+    public double Num1()
+    {
+        return FirstNumber = _validateInput.ValidateInput();
+    }
+
+    public double Num2()
+    {
+        return SecondNumber = _validateInput.ValidateInput();
```

Comment:
why wouldn't do a validation outside of the class? In this case we won't need `Num1`, `Num2`, `DoAdd` methods